source/Guides/Real-Time-Operating-System-Setup/Real-Time-Linux/build_rt_kernel_using_docker.md

```diff
@@ -2,7 +2,16 @@
 
 ## Introduction
 
-Ubuntu 20.04 x86_64 docker container is used to cross-compile a new kernel. There is a Dockerfile which can be used for that purpose. If you want to build it using gitpod you need to run https://gitpod.io/#https://github.com/ros-realtime/rt-kernel-docker-builder. It will spawn a docker container automatically for you.
+This document explains how to build a real-time kernel using a docker container provided by the ROS Real-Time Working Group. The docker container comes with cross-compilation tools installed, and a ready-to-build RT kernel. This should be the prefered option for those users who simply want to use to cross-compile a new kernel.
```

Comment:
Do we need to explicitly mention here that it is a description of how to cross-compile RT_PREEMPT Kernel for Raspberry PI4?
should we describe a workflow that people understand what we are doing and why? E.g. that we take kernel apply RT patches, add configuration, and then build it?

Reply:
@razr  @LanderU and I tough that it would be useful to update the docker image to support multiple architectures. This way this guide would work for any platform and not just for the RPI4. Depending on what we want to do I would choose these options:
1. If we prefer to keep it simple we could provide this guide just for the RPI4. 
2. If we want to support multiple platforms we could update the the tool and a note in this guide explaining only RPI4 is supported for the moment.